coding-standards/html.md

```diff
@@ -1,10 +1,359 @@
-# HTML development standards
-This contains HTML standards.
 
 
-## Coding conventions
-https://google.github.io/styleguide/htmlcssguide.html
+# HTML Development Standards
 
+## Coding Conventions
 
-## Enforcing tool and config
-- TODO
+[Google HTML/CSS Style Guide](https://google.github.io/styleguide/htmlcssguide.html)
+
+## Additional Conventions
+
+# General
+
+## HTML is a Markup Language
```

Comment:
I think instead of heading these can be simple blog text.

Reply:
*bold

Reply:
Updated few guidelines to have bold text and explanation instead.